packages/cli/src/commands/tag/index.ts

```diff
@@ -1,13 +1,15 @@
 import * as git from "@changesets/git";
 import { getPackages } from "@manypkg/get-packages";
 import { log } from "@changesets/logger";
+import { filterTaggablePackages } from "../../utils/versionablePackages";
+import { Config } from "@changesets/types";
 
-export default async function run(cwd: string) {
+export default async function run(cwd: string, config: Config) {
```

Comment:
```suggestion
export default async function tag(cwd: string, config: Config) {
```

and the same change could be made to `publish` (+ status command could get use `status` and not `getStatus`)